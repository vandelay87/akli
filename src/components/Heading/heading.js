import React from 'react';

const Heading = ({ title, size, align }) => (
  <h1>{`${title}!!!`}</h1>
);

export default Heading;
