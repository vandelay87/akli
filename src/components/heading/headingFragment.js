import { graphql } from 'gatsby'

export const query = graphql`
  fragment Heading on ContentfulHeading {
    id
    title
    size
    align
  }
`
