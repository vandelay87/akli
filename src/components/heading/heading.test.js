import renderer from 'react-test-renderer';
import { title, size, align } from './heading.stories';

describe('heading.js', () => {
  it('renders component', () => {
    const titleTree = renderer
      .create(title())
      .toJSON();
    const sizeTree = renderer
      .create(size())
      .toJSON();
    const alignTree = renderer
      .create(align())
      .toJSON();

    expect(titleTree).toMatchSnapshot();
    expect(sizeTree).toMatchSnapshot();
    expect(alignTree).toMatchSnapshot();
  });
});
