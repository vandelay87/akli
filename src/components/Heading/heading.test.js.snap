// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`heading.js renders component 1`] = `
<h1
  className="sc-bdVaJa vmyRs"
  size={1}
>
  Hello world
</h1>
`;

exports[`heading.js renders component 2`] = `
<h3
  className="sc-bdVaJa bOQwgh"
  size={3}
>
  h3 heading
</h3>
`;

exports[`heading.js renders component 3`] = `
<h1
  className="sc-bdVaJa dFHsyD"
  size={1}
>
  center align
</h1>
`;
